dns/opnblock/src/opnsense/mvc/app/models/OPNsense/opnblock/opnblock.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace OPNsense\opnblock;
+use OPNsense\Base\BaseModel;
+class OpnBlock extends BaseModel
```

Comment:
Not sure if `OpnBlock` will work when everywhere else it's just lowercase. I'd suggest Opnblock so you also wont get trouble with Actions.